docs/reference/docs/reindex.asciidoc

```diff
@@ -484,6 +484,18 @@ Use in conjunction with `max_docs` to control what documents are reindexed.
 Set to a list to reindex select fields. 
 Defaults to `true`. 
 
+`type`:::
+(Optional, string)
+deprecated:[6.0.0,Types are deprecated and in the process of being removed. See <<removal-of-types>>.]
+<<mapping-type-field,Document type>> to reindex.
++
+[WARNING]
+====
+By default, all reindexed documents are assigned the `_doc` document type. To
```

Comment:
The original problem here really was that the user expected an error. I think that is somewhat lost in this new version and I prefer the original text (here) instead. WDYT?

```suggestion
Types in source indices are always ignored, also when not specifying a
destination `type`. If explicitly specifying destination `type`, the specified
type must match the type in the destination index or be either unspecified or
the special value `_doc`. See <<removal-of-types>> for further details.
```

Reply:
👍  That works for me.